### ISSUE TITLE
Input Components File returns a temporary file object with a random ending

### ISSUE THREAD

**Issue by DavidZha1994** [gradio-app/gradio#360]
After upgrading from gradio-2.3.6 to gradio-2.3.6, random ending of file object is not beginning with "_" anymore,This makes it hard for me to separate it from the original name of the file.

**Comment by abidlabs** [comment]
@aliabid94 it's possible to get the original name in a different way, right?

**Comment by DavidZha1994** [comment]
any updates?

**Comment by piegu** [comment]
Hi, I've still the same question. 

How to get the name of the input file and not the one of the temporary file created by Gradio?

The following code from a pdf file named "example.pdf" gives for example "/tmp/example958q5pem.pdf", not "example.pdf".

```
import gradio as gr

def get_filename(file_obj):
    return file_obj.name

gr.Interface(get_filename, "file", "text").launch()
```

**Comment by aliabid94** [comment]
Hmm sorry for the late response, currently we automatically append the hash. Will reopen this issue to fix this.

**Comment by DoctorXK** [comment]
> Hmm sorry for the late response, currently we automatically append the hash. Will reopen this issue to fix this.

Hi there, 

I think having access to the original filename is a very signficant property which will be frequently used. Hope you can fix this soon :)

**Comment by abidlabs** [comment]
Sorry for the long wait, but this will be fixed by #2073. In Gradio 3.2, you will be able to do `file_obj.orig_name` to get the original uploaded file name.

**Comment by Seedmanc** [comment]
After the breaking change to the Files component involving uploading, the orig_name now returns path instead of just name. How to get the name again?

**Comment by abidlabs** [comment]
Ah thanks for notifying us @Seedmanc, will reopen this issue to get this fixed.

**Comment by Redcof** [comment]
> Sorry for the long wait, but this will be fixed by #2073. In Gradio 3.2, you will be able to do `file_obj.orig_name` to get the original uploaded file name.

Thanks @abidlabs
